src/lib/select/select.spec.ts

```diff
@@ -2690,8 +2693,74 @@ describe('MdSelect', () => {
 
   });
 
-});
+  describe('compareWith behavior', () => {
+    let fixture: ComponentFixture<NgModelCompareWithSelect>;
+    let instance: NgModelCompareWithSelect;
+
+    beforeEach(async(() => {
+      fixture = TestBed.createComponent(NgModelCompareWithSelect);
+      instance = fixture.componentInstance;
+      fixture.detectChanges();
+    }));
+
+    it('should have a selection', () => {
```

Comment:
Nit: Can you wrap these two tests in a describe that says 'when comparing by value`, so it's clear what comparator is being used?